### PR TITLE
Release/v2.0.1

### PR DESCRIPTION
**Pull request by xross**
Candidate 2.0.1 release

### CODE REVIEW

File: lib_xud/src/core/XUD_Signalling.xc [hunk 1]
```diff
@@ -69,155 +59,46 @@ int XUD_Init()
                 /* Shouldn't expect to get here, but ignore anyway */
                 break;
         }
-
-#else
```
Comment (mbanth) [inline lib_xud/src/core/XUD_Signalling.xc:73]: I'm a bit surprised that `XUD_Init()` does not contain any XS2-specific code.  Is that intentional or an oversight?

**Review by mbanth — COMMENTED** [review]
One question, otherwise it looks good as far as I can tell.

File: lib_xud/src/core/XUD_Signalling.xc (hunk 1 above)
Comment (xross) [reply lib_xud/src/core/XUD_Signalling.xc:73]: HI @mbanth most differences between archs have now been moved to a Hardware Abstraction Layer (see XUD_HAL.xc)

This aids code readability and hopefully simplifies adding new archs in the future.

Comment (mbanth) [reply lib_xud/src/core/XUD_Signalling.xc:73]: Sounds good, @xross.  Thanks for the explanation.

**Review by mbanth — APPROVED** [review]

**Pull request merged**